### PR TITLE
working version of #12282

### PR DESCRIPTION
**Pull request by ghost**
it just werks

the most notoriously bad turf in the game for update icon..
now doesn't have an update icon. or an initialise. In fact, it's super fucking fast at loading. Decreases loading time of server massively, and my god it just werks

clickcatcher works with SEE_BLACKNESS now after the addition of: "click traffic control"- a planesmaster on the darkness plane with mouse opacity set so that the click goes through and is caught by the OG click catcher

also changes the planes and layers file a little

### CODE REVIEW

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -4,6 +4,10 @@
 /mob
 	plane = MOB_PLANE
 
+proc/what_is_see_blackness()
+	to_chat(world,"[SEE_BLACKNESS]")
+	return SEE_BLACKNESS
```
Comment (ghost) [inline code/modules/mob/mob.dm]: ahahhaha debug text I'll remove

**Comment by ghost** [comment]
ALL POSSIBLE BECAUSE OF 🛫🛩️🛬✈️😍😩👌🏻💦🍆

**Comment by SonixApache** [comment]
#12282 of course being...

**Comment by Zennyization** [comment]
He does it again, the absolute madman.

**Comment by ghost** [comment]
Don't merge this one just yet; I came up with an idea of how this might not work in one small situation which should be fixed easily but haven't even tested if it's broken yet

**Comment by clusterfack** [comment]
:airplane_arriving: :boom:

**Comment by ghost** [comment]
alright the issue I thought might be an issue has been fixed. 👍 all good here

**Comment by ghost** [comment]
okay I fixed the shitcode that I did because I felt bad about it but now this is 100% working and 100% ready to merge!

File: vgstation13.dme [hunk 2]
```diff
@@ -2274,7 +2274,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```
Comment (jknpj) [inline vgstation13.dme]: every time

Comment (ghost) [reply vgstation13.dme]: HECK

File: code/modules/mob/hearing/virtualhearer.dm [hunk 3]
```diff
@@ -79,6 +79,10 @@ var/list/stationary_hearers = list(	/obj/item/device/radio/intercom,
 		sight |= adding
 	if(removing)
 		sight &= ~removing
+	if(sight & (SEE_TURFS|SEE_MOBS|SEE_OBJS))
```
Comment (ghost) [inline code/modules/mob/hearing/virtualhearer.dm]: this was the shitcode, too. It was a little too hacked together for my tastes, even for our codebase.

**Comment by ghost** [comment]
wait hold on, lemme check another thing.

**Comment by ghost** [comment]
okay that's my final change it works perfectly and planes are the best also if this gets merged we can have a dorf fort map

File: code/modules/mining/mine_turfs.dm [hunk 4]
```diff
@@ -38,122 +39,28 @@
...
+	else
+		overlays += rock_overlay
+		icon_state_to_appearance[icon_state] = appearance
+
 
 turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_lighting_update = 0, var/allow = 1)
 	mineral_turfs -= src
 	return ..(N, tell_universe, 1, allow)
 
-/turf/unsimulated/mineral/initialize() 	// TODO: OPTIMISE THIS USING PLANES
```
Comment (ghost) [inline code/modules/mining/mine_turfs.dm:51]: toDONE

**Comment by ghost** [comment]
Cluster (the only collaborator as far as I know who has experience in planes and planesmasters) reviewed this PR 
image
^ both of these are true.

**Comment by ghost** [comment]
If you want an idea of how much faster this makes the turfs update; set base turf on box station to /turf/unsimulated/mineral - Previously this would take a LOOOOOOOOOOOOOONNNNNNNNNNNNG time; and kill the server. With this change, it took 1 minute. ONE.

**Comment by ghost** [comment]
don't merge it lost a couple minutes work

**Comment by ghost** [comment]
There's no longer a secret repo conflict, please remove the tag.
Also absolutely 100% works with all sight flag variations and the click catcher works too.

**Pull request merged**